zenstruck/foundry/2.6/src/Fixtures/AppStory.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Fixtures;
```

Comment:
nor sure which namespace to use

Reply:
Since `make:story` creates in `App\Story`, maybe we should match?

Reply:
yes you're right

Reply:
done